src/e2e-test/java/io/cdap/plugin/bigquery/stepsdesign/BigQueryBase.java

```diff
@@ -248,16 +279,60 @@ public void verifyTheBigQueryValidationErrorMessageForInvalidProperty(String pro
     Assert.assertEquals(expectedColor, actualColor);
   }
 
+  @Then("Verify the BigQuery validation error message for reference name {string}")
```

Comment:
Why are we duplicating the step definition. I already informed and mentioned long back to go through the existing tests and use the existing steps. Here as well, no need to create this step definition again. You can use the existing  in-line error message verify step. And also I don't think you are using this in your scenario.